.travis.yml

```diff
@@ -23,15 +23,55 @@ addons:
 
 jobs:
   include:
-    - script: mkdir -p build && cd build && cmake -D EXAMPLES=ON -D MPI=OFF -D ICB=OFF               .. && make VERBOSE=1 && make test;
-    - script: mkdir -p build && cd build && cmake -D EXAMPLES=ON -D MPI=ON  -D ICB=OFF               .. && make VERBOSE=1 && make test;
-    - script: mkdir -p build && cd build && cmake -D EXAMPLES=ON -D MPI=OFF -D ICB=ON                .. && make VERBOSE=1 && make test;
-    - script: mkdir -p build && cd build && cmake -D EXAMPLES=ON -D MPI=ON  -D ICB=ON                .. && make VERBOSE=1 && make test;
-    - script: mkdir -p build && cd build && cmake -D EXAMPLES=ON -D MPI=ON  -D ICB=ON -DCOVERALLS=ON .. && make VERBOSE=1 && make test;
-    - script: ./bootstrap && ./configure                                                                && make VERBOSE=1 && make check && make distcheck;
-    - script: ./bootstrap && ./configure --enable-mpi                                                   && make VERBOSE=1 && make check && make distcheck;
-    - script: ./bootstrap && ./configure              --enable-icb                                      && make VERBOSE=1 && make check && make distcheck;
-    - script: ./bootstrap && ./configure --enable-mpi --enable-icb                                      && make VERBOSE=1 && make check && make distcheck;
+    # precise <=> test "older" systems, without ICB, without cmake (too old to be supported)
+    - stage: precise-autotools-seq
+      dist: precise
+      script: ./bootstrap && ./configure                                                                && make VERBOSE=1 && make check && make distcheck;
+    - stage: precise-autotools-mpi
+      dist: precise
+      script: ./bootstrap && ./configure --enable-mpi                                                   && make VERBOSE=1 && make check && make distcheck;
+    # trusty <=> test "older" systems, without ICB
+    - stage: trusty-cmake-seq
+      dist: trusty
+      script: mkdir -p build && cd build && cmake -D EXAMPLES=ON -D MPI=OFF -D ICB=OFF               .. && make VERBOSE=1 && make test  && make package_source;
```

Comment:
CMake "make package_source" <=> autotools "make dist" only, **not** "make dist**check**". You may drop that ?